glob.go

```diff
@@ -57,3 +62,14 @@ func Glob(pattern, subj string) bool {
 	// All parts of the pattern matched
 	return true
 }
+
+// globSequence reports if pattern consits of glob characters only.
+// I.e., it returns true for "", "*", "**", "***", etc., and false for all other patterns.
+func globSequence(pattern string) bool {
+	for i := 0; i < len(pattern); i++ {
+		if pattern[i] != '*' {
```

Comment:
Should this be `pattern[i] != GLOB`? See how `const GLOB` is used in other places in this file.